test/env/compose.go

```diff
@@ -0,0 +1,97 @@
+package env
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+
+	nadv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+
+	kubevirtv1 "kubevirt.io/api/core/v1"
+)
+
+func ComposeLayer2WithSubnetNAD(namespace string) *nadv1.NetworkAttachmentDefinition {
```

Comment:
I don't follow the compose prefix. 

What are you composing ? I would expect generate or something similar.

Reply:
I will move this to generate.go

Reply:
now we have this twice isnt it ?
both here and on generate

Reply:
what about this one ? if we want we can fix it on a follow-up